gradle/publish.gradle

```diff
@@ -42,16 +44,36 @@ bintray {
         repo = 'releases'
         name = 'atlasdb'
         licenses = ['BSD-3-Clause']
-        publications = ['bintray']
+        publications = ['artifactory']
     }
 }
 
 bintrayUpload.onlyIf {
     System.getenv('BINTRAY_USERNAME') && System.getenv('BINTRAY_PASSWORD') && project.version ==~ /\d+\.\d+\.\d+(-alpha|-beta)?(\+\d{3})?/
 }
 
-bintrayUpload.dependsOn { generatePomFileForBintrayPublication }
+bintrayUpload.dependsOn { generatePomFileForArtifactoryPublication }
 bintrayUpload.dependsOn { sourceJar }
 bintrayUpload.dependsOn { testJar }
 bintrayUpload.dependsOn { build }
 
+artifactory {
+  publish {
+    contextUrl = 'https://oss.jfrog.org/artifactory/'
+    repository {
+      repoKey = 'oss-snapshot-local'
+      username = System.env.ARTIFACTORY_USERNAME
+      password = System.env.ARTIFACTORY_PASSWORD
+      maven = true
+    }
+    defaults {
+      publications ('artifactory')
+      properties = ['git': project.version.toString()]
+    }
+  }
```

Comment:
Done by the publication defined above in the pom.  Not sure if we need to add it anywhere else.